src/components/ReportActionItem/IOUPreview.js

```diff
@@ -159,7 +162,10 @@ const IOUPreview = (props) => {
                                     <View style={styles.iouPreviewBoxAvatar}>
                                         <MultipleAvatars
                                             icons={[managerAvatar, ownerAvatar]}
-                                            secondAvatarStyle={[styles.secondAvatarInline]}
+                                            secondAvatarStyle={[styles.secondAvatarInline, {borderColor: themeColors.cardBG},
+                                                props.isFocused
+                                                    ? {borderColor: themeColors.border}
+                                                    : undefined]}
```

Comment:
No inline styles. We can move these to styles.js 

A clean syntax for conditional styles will be 
```
[
	props.isFocused && {borderColor: themeColors.border}
]
```